.eslintrc.json

```diff
@@ -53,8 +53,15 @@
                 "ignoreRestSiblings": true
             }
         ],
-        "@typescript-eslint/interface-name-prefix": ["error", { "prefixWithI": "never" }],
-        "curly": ["error", "all"]
+        "curly": ["error", "all"],
+        "comma-dangle": "off",
+        "@typescript-eslint/comma-dangle": "off",
+        "@typescript-eslint/naming-convention": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "@typescript-eslint/ban-ts-comment": "off",
+        "@typescript-eslint/dot-notation": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/ban-types": "off"
```

Comment:
@antipin @eszthoff what do you think guys about these rules?

Reply:
@mukiienko 

@typescript-eslint/naming-convention, @typescript-eslint/dot-notation - was added with a new version of eslint-config-airbnb-typescript

@typescript-eslint/explicit-module-boundary-types, @typescript-eslint/ban-ts-comment  - was added with a new version of typescript-eslint

Reply:
Sorry, I missed this comment. Can you give a quick summary of the impact if we leave these rules on?

Reply:
@eszthoff  Hello, We get errors and warnings @typescript-eslint/naming-convention:
![image](https://user-images.githubusercontent.com/104063210/193277832-bcbdecef-7464-4b22-af49-9220686bda18.png)
@typescript-eslint/dot-notation:
![image](https://user-images.githubusercontent.com/104063210/193277982-b6b99c36-5eb4-447f-9d24-4b71f496eaf1.png)
@typescript-eslint/explicit-module-boundary-types:
![image](https://user-images.githubusercontent.com/104063210/193278561-9a5c76a9-6125-466f-88ad-41ee0dbe27a6.png)
@typescript-eslint/ban-ts-comment:
![image](https://user-images.githubusercontent.com/104063210/193278759-c5aaa3a2-4c16-4ac0-9fbf-cd73df7970e5.png)

Reply:
Thanks for this!

* typescript-eslint/naming-convention - seems to happen only in storybook, so I suggest to ignore this rule only there (`/stories/.eslintrc.json`)
* typescript-eslint/dot-notation - I don't see any errors related to that. What am I missing?
* typescript-eslint/explicit-module-boundary-types - it's looks ok for me to turn it off. But maybe in the future we could fix these, so that we have stronger type guards. If others agree with that, maybe you can make a followup ticket in OneUI.
* typescript-eslint/ban-ts-comment - I'm wondering about that. I think it is ok to have a warning on these, as indeed we should not ignore TS. There are a few exceptions to that rule, when TS is indeed at fault. I think in those cases, we can ignore them specifically in the file itself. What do you think?

Reply:
@eszthoff about it also exists in:
![image](https://user-images.githubusercontent.com/104063210/193286477-4fc1022b-1a41-4d44-82cd-c31cff194a36.png)
about typescript-eslint/dot-notation it was second screenshot

About  typescript-eslint/ban-ts-comment  you mean add to these files ignore ?

Reply:
@PavelKirvelTextkernel I don't understand what you referring to with `about it also exists in:`.

> about typescript-eslint/dot-notation it was second screenshot
I don't see errors related to dot-notation in that screenshot

> About typescript-eslint/ban-ts-comment you mean add to these files ignore
I meant that we should leave the error on, and turn them off locally in the files where they are really needed. But maybe other people have a different idea.

Reply:
That makes sense, but that is the exception. So I think we should leave the rule in general and only disable it for that one line, using `// eslint-disable(-next)-line @typescript-eslint/dot-notation`. How does that sounds to you?

Reply:
It sounds good